python/tvm/autotvm/tophub.py

```diff
@@ -55,7 +55,7 @@
     'mali':             "v0.05",
     'intel_graphics':   "v0.01",
 
-    'vta':              "v0.06",
+    'vta':              "v0.07",
```

Comment:
cuda and arm cpu version also need to be updated

Reply:
done